pkg/prompts/prompts.go

```diff
@@ -127,6 +143,65 @@ func validateBiggerThanZero(input string) error {
 	return nil
 }
 
+func (r *realPrompter) CaptureListDecision(
+	prompter Prompter, // we need this in order to be able to run mock tests
```

Comment:
think the prompter can be avoided

Reply:
I am not sure it can though. See, the implementation is on `realPrompter` but to mock it we need to pass the mock for the other prompts required in this function.